modules/document_repository/test/document_repositoryTest.php

```diff
@@ -127,54 +124,25 @@ function testDocumentRepositoryDoespageLoad()
         $bodyText = $this->webDriver->findElement(
             WebDriverBy::cssSelector("body")
         )->getText();
-        $this->assertRegexp("/Upload File/", $bodyText);
+        $this->assertContains("TESTTESTTESTTEST", $bodyText);
```

Comment:
This looks like it is left over debugging

Reply:
If this is resolved I can approve this PR.

Reply:
@HenriRabalais This one is for the test case, I inserted a test data "TESTTESTTESTTEST" into the table.

Reply:
$this->DB->insert(
            "document_repository_categories",
            array(
             'id'            => '9999999',
             'category_name' => 'TESTTESTTESTTEST',
             'parent_id'     => '0',
             'comments'      => 'Test Comment',
            )
        );

Reply:
@HenriRabalais Yeah I think this is necessary for the test code and should stay.